sdk/src/dataMapper/PortalDataMapper.ts

```diff
@@ -47,8 +47,46 @@ export default class PortalDataMapper extends BaseDataMapper<Portal> {
     return hash;
   }
 
-  async bulkAttest() {
-    throw new Error("Not implemented");
+  async simulateBulkAttest(
+    portalAddress: Address,
+    attestationPayloads: AttestationPayload[],
+    validationPayloads: string[][],
+  ) {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const attestationPayloadsArg: any[] = [];
```

Comment:
Let's avoid using `any`.
Maybe we can create a type to cover this specific case?